### PR TITLE
chore: rename `local run` -> `run local`

### PR DESCRIPTION
**Pull request by dannyrandall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/run_local.go [hunk 1]
```diff
@@ -640,16 +640,15 @@ func (o *localRunOpts) getSecret(ctx context.Context, valueFrom string) (string,
...
-func BuildLocalRunCmd() *cobra.Command {
-	vars := localRunVars{}
+// BuildRunLocalCmd builds the command for running a workload locally
+func BuildRunLocalCmd() *cobra.Command {
+	vars := runLocalVars{}
 	cmd := &cobra.Command{
-		Use:    "local run",
-		Short:  "Run the workload locally.",
-		Long:   "Run the workload locally.",
-		Hidden: true,
```
Comment (Lou1415926) [inline internal/pkg/cli/run_local.go:650]: 👋🏼

File: cmd/copilot/main.go [hunk 2]
```diff
@@ -81,7 +81,7 @@ func buildRootCmd() *cobra.Command {
 	cmd.AddCommand(cli.BuildSvcCmd())
 	cmd.AddCommand(cli.BuildJobCmd())
 	cmd.AddCommand(cli.BuildTaskCmd())
-	cmd.AddCommand(cli.BuildLocalRunCmd())
+	cmd.AddCommand(cli.BuildRunLocalCmd())
```
Comment (Lou1415926) [inline cmd/copilot/main.go:84]: I think the help menu will show up like
```
$ copilot -h
Development
   run     Run workloads locally
```

People _might_ (or might now) be confused about its relationship with the existing `task run` and `job run`, which is probably fine.

**Review by Lou1415926 — APPROVED** [review]

File: cmd/copilot/main.go (hunk 2 above)
Comment (dannyrandall) [reply cmd/copilot/main.go:84]: i think that's a fair concern; hopefully our descriptions are clear enough:
```
$ copilot task run -h
Run a one-off task on Amazon ECS.

$ copilot job run -h
Invoke a job in an environment.
```

**Review by huanjani — APPROVED** [review]
LGTM

**Pull request merged**